testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -343,6 +346,9 @@ public void getTableComplex() throws IOException {
     assertThat(columnDefinition)
         .usingRecursiveComparison()
         .isEqualTo(new ColumnDefinition("col1", "frozen<map<date, varchar>>", false));
+
+    // but also check we seem to have right number of columns as well
+    assertThat(table.getColumnDefinitions()).hasSize(4);
```

Comment:
why not combining this all together at line 339:

```
assertThat(table.getColumnDefinitions())
   .isNotNull()
   .isNotEmpty()
   .hasSize(4)
```

although only `hasSize` would be enough and you can combine column assertion:


```
assertThat(table.getColumnDefinitions())
   .hasSize(4)
   .anySatisfy(columnDefinition -> assertThat(columnDefinition)
        .usingRecursiveComparison()
        .isEqualTo(new ColumnDefinition("col1", "frozen<map<date, varchar>>", false)
  );
```

Reply:
Thanks! Will change along suggestion. One possible reason to keep some checks separate is for more granular detection (which of checks failed), but hopefully line number matches failing part of assertion (or assertion fail message).